shared/actions/login/index.js

```diff
@@ -74,7 +74,7 @@ function getAccounts (): AsyncAction {
 
 export function login (): AsyncAction {
   return (dispatch, getState) => {
-    startLoginFlow(dispatch, getState, enums.provisionUi.ProvisionMethod.device, 'Keybase username', 'lorem ipsum', Constants.loginDone)
+    startLoginFlow(dispatch, getState, Constants.loginDone)
```

Comment:
:sheep: This doesn't follow our AsyncAction flow. Instead of passing in `dispatch` and `getState` you should just make `startLoginFlow` an async action and call it like: `dispatch(startLoginFlow(Constants.loginDone))`